std/log/handlers.ts

```diff
@@ -99,11 +102,14 @@ interface FileHandlerOptions extends HandlerOptions {
 }
 
 export class FileHandler extends WriterHandler {
+  protected _buff!: BufWriterSync;
   protected _file!: File;
   protected _filename: string;
   protected _mode: LogMode;
   protected _openOptions: OpenOptions;
   #encoder = new TextEncoder();
+  #queue: Queue<string> = new Queue<string>();
+  #drainComplete!: Deferred<void>;
```

Comment:
I think logging output should be done synchronously. Is this patch making that not so?

Reply:
@ry Ultimately, all logs are being  output to file synchronously (through `BufWriterSync.writeSync()`).  The queue here is used as a buffer for log messages to be processed in the next event loop.  I'm actually going to close this PR.  I've just had an epiphany on how to radically simplify this solution and achieve the same result.